.github/workflows/sync-lockfiles.yml

```diff
@@ -94,8 +103,14 @@ jobs:
         # NOTE: If there is a pending PR, this action will simply update it with a forced push.
         uses: peter-evans/create-pull-request@v6
         with:
-          title: Sync lockfile with Zenoh's
-          body: Automated synchronization of the Cargo lockfile with Zenoh. This is done to ensure plugin ABI compatibility.
+          title: Sync Cargo lockfile with Zenoh's
```

Comment:
It would be nice to have the date and commit in the title so it's easier to check and backtrack sync points.
E.g., look at how PRs are displayed in zenoh-c: https://github.com/eclipse-zenoh/zenoh-c/pulls?q=is%3Apr+is%3Aclosed+sync

Reply:
The title is not the squash commit message, that is defined in line 128. I did not put the info in the pull request title because it would be very noisy.

Reply:
The commit message will be of the form "Sync `Cargo.lock` with eclipse-zenoh/zenoh@**zenoh-head-hash** from **zenoh-head-date** (#**pull-request-number**)"

Reply:
Again, if you look at this page https://github.com/eclipse-zenoh/zenoh-c/pulls?q=is%3Apr+is%3Aclosed+sync what you see is the PR title, not the commit message.

Reply:
That is because the squash/merge commit message was previously not defined. It it now.

Even when a GitHub user attempts to merge a pull request, the UI ask them for a commit message, which _defaults_ to the pull request title.

This is done using the `--subject` option (i.e. Subject text for the merge commit) in the `gh pr merge` in line 128.